common/src/main/resources/assets/rrls/lang/it_it.json

```diff
@@ -0,0 +1,12 @@
+{
+  "text.autoconfig.rrls.title": "Remove Reloading Screen",
```

Comment:
Add the localization for this line as follows https://github.com/dimadencep/RemoveReloadingScreen/blob/1.20.x/dev/common/src/main/resources/assets/rrls/lang/ru_ru.json#L2.